config/roles.yml

```diff
@@ -262,6 +262,8 @@ ml_read_access:
     - 'cluster:admin/opensearch/ml/models/search'
     - 'cluster:admin/opensearch/ml/tasks/get'
     - 'cluster:admin/opensearch/ml/tasks/search'
+    - 'cluster:admin/opensearch/ml/connectors/get'
```

Comment:
Hi @zane-neo , could you please fix permissions order? All permissions need to sorted. Otherwise LGTM

Reply:
@willyborankin can you elaborate more on this? I have no idea what is the correct permission order. Thanks.

Reply:
@willyborankin I've changed the order of the permission list, please take a look, thanks.